docs/content/reference/cli/glooctl_debug.md

```diff
@@ -36,6 +42,9 @@ glooctl debug [flags]
 ### SEE ALSO
 
 * [glooctl](../glooctl)	 - CLI for Gloo
+* [glooctl debug envoy](../glooctl_debug_envoy)	 - Dump envoy admin interface to a local directory (requires Gloo running on Kubernetes)
```

Comment:
Would this be for a particular gateway proxy? all gateway proxies? If so, maybe add that
```suggestion
* [glooctl debug envoy](../glooctl_debug_envoy)	 - Dump information from the Envoy admin interface for a gateway proxy to a local directory (requires Gloo running on Kubernetes)
```